model-optimizer/unit_tests/extensions/middle/ConvertGroupedStridedSlice_test.py

```diff
@@ -60,8 +60,8 @@
     'concat_1_data': {'value': None, 'shape': None, 'kind': 'data'},
 
     'op_output': {'kind': 'op', 'op': 'Result'},
-    'op_output_1': {'kind': 'op', 'op': 'Result'},
-    'op_output_2': {'kind': 'op', 'op': 'Result'},
+    'op_output_1': {'kind': 'op', 'op': 'Result', 'keep_output_port': True},
+    'op_output_2': {'kind': 'op', 'op': 'Result', 'keep_output_port': True},
 
```

Comment:
also, add case with 'keep_output_port' equal to `False`

Reply:
See my comment above.

Reply:
Case with `keep_output_port` value equal to `False` is not applicable to this function. And test for `add_opoutput` with default value `keep_output_port` = `False` already exist.